ketcher-autotests/tests/Settings/General/default-settings-verification.spec.ts

```diff
@@ -1,21 +1,64 @@
-import { test } from '@playwright/test';
+import { Page, test } from '@playwright/test';
 import {
   selectTopPanelButton,
   TopPanelButton,
   takeEditorScreenshot,
   waitForPageInit,
+  drawBenzeneRing,
+  pressButton,
+  copyAndPaste,
 } from '@utils';
 
+async function generalDefaultSettings(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  const deltaX = 0;
+  const deltaY = 150;
+  const anyX = 638;
+  const anyY = 524;
+  await page.mouse.move(anyX, anyY);
+  await page.mouse.wheel(deltaX, deltaY);
```

Comment:
seems these codes are also the same as the previous manipulations

Reply:
I can move it to a separate function, but it's still a temporary one.